app/Http/Controllers/ContactController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ContactRequest;
+use App\Mail\NewReport;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+
+class ContactController extends Controller
+{
+    public function show()
+    {
+        return view('contacts');
+    }
+
+    public function store(ContactRequest $request)
+    {
+        $mail = new NewReport(
```

Comment:
у нас про форму обратной связи и контакты
а не про отчеты/жалобы
```suggestion
        $mail = new NewContact(
```